graphql/resolvers.ts

```diff
@@ -1,23 +1,24 @@
 import { login, logout } from '../helpers/controllers/authController'
 import db from '../helpers/dbload'
 
-const { Lesson, User } = db
+const { Lesson } = db
 
 export default {
   Query: {
     lessons() {
       return Lesson.findAll({
-        include: [
-          'challenges',
-          {
-            model: User,
-            through: {
-              attributes: ['isPassed', 'isTeaching', 'isEnrolled']
-            }
-          }
-        ],
+        include: ['challenges'],
```

Comment:
When querying Lessons, we should get all the information about whether the User has enrolled / passed / teaching the lesson.  This way the information can be rendered in the UI.

Otherwise the UI has no way to tell whether the User is done with the lesson or not.